com.unity.render-pipelines.high-definition/Documentation~/HDRP-Features.md

```diff
@@ -167,14 +167,43 @@ For more information, including the full list of light properties, see the [Ligh
 
 HDRP supports cubemap and planar GPU Reflection Probes to help you produce realistic reflections in your Scene in real time. For more information, see the [introduction to Reflection Probes](Reflection-Probes-Intro.md)
 
+### Planar reflection probe filtering
+
+![](Images/PlanarReflectionFiltering-Feature.png)
+
+Planar reflection probe filtering combines the result of planar reflection and surfaces smoothness. HDRP includes accurate filtering for planar reflection that is close to being physically-based.
```

Comment:
Let's reword it like this: (no need for "filtering" )

We will remove planar reflection from the previous section:

HDRP supports cubemap reflection taking into account surface smoothness to help you produce realistic reflections in your Scene in real time. For more information, see the [introduction to Reflection Probes](Reflection-Probes-Intro.md)

 ### Planar reflection probe

=> HDRP supports Planar reflection probe taking into account surfaces smoothness, allowing to mimic mirror to rough floor. HDRP includes accurate filtering for planar reflection that is close to being physically-based.

Reply:
and I add a link to this pages: https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@10.0/manual/Planar-Reflection-Probe.html